### DISCUSSION TITLE
Tabla lugar bonus

### DISCUSSION THREAD

**Discussion by antoniablanco** [IIC2133-PUC/2024-1#141] [Tarea 2]
Hola, ¿será posible ir viendo los resultados parciales de las tablas de posiciones del bonus, para saber como uno se encuentra con respecto al resto?

**Comment by benjavicente** [comment]
Hola!

Tener un servidor con el ranking sería difícil. Por problemas técnicos del DCC, perdimos todo el setup que tenía el servidor.
Voy a ver si alcanzo eso si a publicar resultados parciales, pero no me puedo comprometerme con nada.

Muy de antemano, esperaría que los tiempos de la parte 1 estén bajo 1 segundo, y de la parte 3, bajo 2~3 segundos (en los test hards).
